spec/menu_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'menu'
+
+describe Menu do
```

Comment:
You could define all of the menu options as constant in your menu class in order to avoid listing everything in your #print test.